GDCpp/scripts/CopyHeadersToGD.bat

```diff
@@ -24,7 +24,7 @@ IF NOT EXIST "%MINGW_DIR%" set MINGW_DIR="C:\MinGW32"
 IF NOT EXIST "%MINGW_DIR%" set MINGW_DIR="C:\MinGW"
 
 IF EXIST "%MINGW_DIR%" (
-	xcopy "%MINGW_DIR%" "%destDir%\MinGW32\" /E /D /Q
+	xcopy "%MINGW_DIR%" "%destDir%\MinGW32\" /E /D /Q /C
```

Comment:
(It's already merged in the master branch, my local master is completely messed because of the pull requests merged with "Squash" on Github)

The xcopy command was failing on an unknown file with "permission denied" on the Windows worker. Ignoring the error doesn't have an impact on the build.